README.md

```diff
@@ -101,11 +113,12 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[X] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
+[X] `$(".post").html()`
+[X] `document.getElementsByClassName("post")[0].innerHTML`
+[X] `document.getElementsByClassName("post").innerHTML`
+// The bottom two will still work in the console but do not use jQuery
 ```
 
```

Comment:
That last one actually won't work because you are calling `.innerHTML` on an array of DOM elements. The brackets are necessary to select a particular element from that array.